packages/web-components/src/components/pricing-table/pricing-table-group.ts

```diff
@@ -9,18 +9,18 @@
 
 import settings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
 import StableSelectorMixin from '../../globals/mixins/stable-selector';
-import DDSStructuredListGroup from '../structured-list/structured-list-group';
+import C4DStructuredListGroup from '../structured-list/structured-list-group';
 import styles from './pricing-table.scss';
 import { carbonElement as customElement } from '../../internal/vendor/@carbon/web-components/globals/decorators/carbon-element';
 const { stablePrefix: ddsPrefix } = settings;
```

Comment:
probably gotta do a massive find and replace of `ddsPrefix` as well

```suggestion
const { stablePrefix: c4dPrefix } = settings;
```